### PR TITLE
update: Setup update playbook to use edge cert api endpoints.

### PR DESCRIPTION
**Pull request by ldjebran**
A step ahead to make the device update repo as private, the update repos already merged and tested to be working.

Fixes # THEEDGE-2489

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Comment by ldjebran** [comment]
Put do not merge, as we need a new signature of the playbook.

**Comment by omaciel** [comment]
image

**Please remember to have the templates signed before merging!**

**Comment by ldjebran** [comment]
Remove do not merge as playbook temple has been signed.

**Review by de1987 — APPROVED** [review]

File: pkg/services/updates.go [hunk 1]
```diff
@@ -8,10 +8,10 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+	url2 "net/url"
```
[the remark this answers is not in the document]
Comment (ldjebran) [reply pkg/services/updates.go]: alias removed.

**Review by AaronH88 — APPROVED** [review]
/lgtm

**Pull request merged**